test/test.js

```diff
@@ -125,7 +125,7 @@ describe('transit', function() {
       });
 
       var roundTrip = recordTransit.fromJSON(recordTransit.toJSON(data));
-      expect(roundTrip).to.eql(data);
+      expect(roundTrip.toString()).to.eql(data.toString());
```

Comment:
This doesn't seem right to me, we want to check deep equality of the objects - not their string representation.

What tests were failing without this? Do you have the latest version of `chai-immutable` installed?